changelog/unreleased/pull-3203

```diff
@@ -0,0 +1,9 @@
+Enhancement: Allow to check specific pack files
+
+Restic check used to check either all packfiles or a subset that is chosen
+either in a deterministic or random way but did not allow to specify the pack
+files to check directly.
```

Comment:
Suggestion:
Restic check used to check either 1) all packfiles or 2) a subset of the packfiles.
The subset was either choosen a) in a deterministic or b) random way.